### PR TITLE
HTML/CSS Quiz - Second Submission

### PR DESCRIPTION
**Pull request by krcummings1**
-Deleted article class names "one, two, three, four".
    -PS, I chose those as the class names because those are also the product names, but thank you for pointing that out. Noted :)
-I think I changed the indentation for the h3 and p elements in a way that you'd approve, but I'm not 100% sure. 
-Took out the padding on the availability element and used text-indent instead. If there was a better way to do this, please let me know.

Hope you're having a safe and fun snow day/weekend! Hope to see you Monday!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML/CSS Quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+
+<body>
+  <article>
```
Comment (krcummings1) [inline index.html]: Took out article class names.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    	</div>
+    	<p class="proddesc">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Doloremque hic iure, est. Quos maiores maxime a fugit voluptates, odit quasi officiis alias minima ex dolor, eos nihil similique fuga nulla.</p>
+    	<p class="avail">
+    		Available
+    	</p>
+    </section>
+	<div class="container">
+	    <section>
+	    	<h3>Specifications:</h3>
+	    		<p>Size: 4" x 4"</p>    <p>Weight: 8 oz.</p>
```
Comment (krcummings1) [inline index.html]: Changed indentation for h3 and p elements, though it looks like the p elements are on the same line here and they weren't supposed to be... Possibly wrapped in a weird way in Sublime due to me sizing the window down.

File: quiz.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+h3 ~ p {
+	font-weight: bold;
+	padding-left: 30px;
+}
+
+.avail {
+	background-color: darkgrey;
+	color: yellow;
+	text-indent: 5px;
```
Comment (krcummings1) [inline quiz.css]: Took out padding and used text-indent to still indent the text, but prevent the element from showing outside the article.

**Comment by chortlehoort** [comment]
Looks good now, Kaylee. Your solution for the padding solved the problem, so that's a good way to do it.  I'll mark this as complete.

:+1:

**Comment by krcummings1** [comment]
Awesome, thanks!

**Pull request merged**